lib/langfuse_sdk.ex

```diff
@@ -68,6 +105,29 @@ defmodule LangfuseSdk do
     LangfuseSdk.Ingestor.ingest_payload(score_event)
   end
 
+  @doc """
+  Get a list of traces.
+
+  ## Parameters
+
+  * `page`: Page number, starts at 1.
+  * `limit`: Limit of items per page. If you encounter api issues due to too large page sizes, try to reduce the limit.
+  * `user_id`: Uncocumented, check API documentation.
+  * `name`: Uncocumented, check API documentation.
+  * `session_id`: Uncocumented, check API documentation.
+  * `from_timestamp`: Optional filter to only include traces with a trace.timestamp on or after a certain datetime (ISO 8601).
+  * `to_timestamp`: Optional filter to only include traces with a trace.timestamp before a certain datetime (ISO 8601).
```

Comment:
@nickgnd, @ArthV usually what I do is just link to the underlying short (eg: see params in the official lib) to avoid repetition. How do you feel about this? (I'm not sure we want to maintain docs in two places)

Reply:
Good point, I think we can add the link to the official documentation.

And maybe says to pass the options as a map (are the options supposed to snake_case? Usually, JS expects camelCase, or Req is doing the conversation?)

Reply:
Thanks, just updated this one to reference the docs. The function accepts an enumerable and the casing can be either snake_case or camelCase (see the link on the body of the function for more details about why).